packages/gasket-plugin-nextjs/lib/prompt.js

```diff
@@ -3,8 +3,23 @@
 /** @type {import('@gasket/core').HookHandler<'prompt'>} */
 module.exports = async function promptHook(gasket, context, { prompt }) {
   const newContext = { ...context };
+
+  if (!('useAppRouter' in context)) {
+    const { useAppRouter } = await prompt([
+      {
+        name: 'useAppRouter',
+        message: 'Do you want to use the App Router? (experimental)',
+        type: 'confirm',
+        default: false
+      }
+    ]);
+
+    newContext.useAppRouter = useAppRouter;
+    newContext.nextServerType = 'defaultServer';
```

Comment:
Wouldn't this line get skipped if `useAppRouter` was already in the context?

Reply:
I added a check outside of the useAppRouter prompt
```
  if (newContext.useAppRouter) {
    newContext.nextServerType = 'defaultServer';
  }
```